llama_stack/apis/common/type_system.py

```diff
@@ -56,6 +71,9 @@ class CustomType(BaseModel):
         JsonType,
         UnionType,
         CustomType,
+        ChatCompletionInputType,
```

Comment:
I think we will likely go back and forth on this design bit once we also get some time with @raghotham in person on Monday but this is OK for now.